photo_points/Controllers/AdminController.cs

```diff
@@ -52,11 +59,28 @@ public IActionResult PhotoStream()
         [HttpGet]
         public IActionResult Pending()
         {
-            return View("Pending");
-            //all the new submissions can be displayed
-            // //  if Capture approval is false. 
-            // // display the Capture image and Data. 
+            //build a view model //
+            PendingViewModel pvm = new PendingViewModel();
+
+           // _adminReviewServices.GetUnapprovedCaptures(); ////the repository is calling the getunapproved captures
 
+            IEnumerable<Capture> pendingImages = _adminReviewServices.GetUnapprovedCaptures();
+            //string mimeType = /* Get mime type somehow (e.g. "image/png") */;
+            //string base64 = Convert.ToBase64String(yourImageBytes);
+            //return string.Format("data:{0};base64,{1}", mimeType, base64);
+
+            //create a foreach loop that goes thru the list and pulls out images that have NOT been approved. 
+            foreach (Capture img in pendingImages) // looking at comments on line 18 and 23 in Capture.cs // should we display all images since they are the default?
```

Comment:
You are correctly using the foreach loop to go through the Captures ( again, I suggest you call each item a "capture" rather than an "img"